bycycle/tests/group/test_features.py

```diff
@@ -0,0 +1,136 @@
+"""Test functions to compute features across epoched data."""
+
+import time
+
+import numpy as np
+
+from pytest import mark, param
+
+from bycycle.group.features import compute_features_2d, compute_features_3d
+
+###################################################################################################
+###################################################################################################
+
+@mark.parametrize("compute_features_kwargs_error", [False, param(True, marks=mark.xfail)])
+@mark.parametrize("compute_features_kwargs_dtype", ['dict', 'list', None])
+def test_compute_features_2d(sim_args, compute_features_kwargs_dtype,
+                             compute_features_kwargs_error):
+
+    n_sigs = 5
+    sigs = np.array([sim_args['sig']] * n_sigs)
+    fs  = sim_args['fs']
+    f_range = sim_args['f_range']
+
+    # return_samples is disregarded when used in compute_features_kwargs.
```

Comment:
Updating.

Which comments? I don't see one here about this, but I remember you left a comment periods in comments in one of Eric's PRs a week or so ago. I typically don't use periods either, not that it really matters though.

Reply:
hahaha, the second line was a joke, as in "did you miss me be overly specific with a ton of comments". :P

I did not at all mean to imply you had not noticed some genuine comment somewhere :)

Reply:
Ohhhh, yeah totally misread that lol.

I'm going to compile a list of your nitpicks, and write a script to parse code and fix those nitpicks automatically. It can one of those automated PR check things :wink: